lib/iris/coords.py

```diff
@@ -1937,11 +1937,12 @@ def _discontiguity_in_bounds(self, rtol=1e-5, atol=1e-8):
         * contiguous: (boolean)
             True if there are no discontiguities.
         * diffs: (array or tuple of arrays)
-            The diffs along the bounds of the coordinate. If self is a 2D
-            coord of shape (Y, X), a tuple of arrays is returned, where the
-            first is an array of differences along the x-axis, of the shape
-            (Y, X-1) and the second is an array of differences along the
-            y-axis, of the shape (Y-1, X).
+            A boolean array or tuple of boolean arrays which are true where
+            there are discontiguities between neighbouring bounds. If self is
+            a 2D coord of shape (Y, X), a tuple of arrays is returned, where
```

Comment:
```suggestion
            a 2D coord of shape (Y, X), a pair of arrays is returned, where
```

Not your fault, as inherited, but I think the use of "tuple" here is unhelpful.

Reply:
addressed in https://github.com/SciTools/iris/pull/4975/commits/6785b40d03016ec42d2539b42c0b3b9ba545829c